delete.js

```diff
@@ -1,25 +1,24 @@
 {
-	let newChattyObj = Object.create(null);
+	let removeMessage = Object.create(null);
 	let deleteButton = document.getElementsByClassName("delete-btn");
 
 	// One IIFE should accept a message element id and then remove the correct element from the DOM.
 	// This IIFE should also remove the corresponding message from the private array that was created
 	// in the previous IIFE.
-	newChattyObj.removeMessage = function(id) {
+	removeMessage.removeFromDOM = function(id) {
 		let currentElement = document.getElementById(id);
 		currentElement.parentNode.remove();
 	}
 
-	newChattyObj.removeFromArray = function(msgArray, id) {
+	removeMessage.removeFromArray = function(msgArray, id) {
```

Comment:
I'm not sure why we would call the object that we are using to overwrite the original chatty object "removeMessage". Seems like it would add confusion to what that object is or what it does.